src/webview_provider/queryResultPanel.ts

```diff
@@ -273,10 +274,7 @@ export class QueryResultPanel implements WebviewViewProvider {
       .get<boolean>("enableNewQueryPanel", true);
     const queryTemplate = workspace
       .getConfiguration("dbt")
-      .get<string>(
-        "queryTemplate",
-        "select * from ({query}) as query limit {limit}",
-      );
```

Comment:
Is there any way you can move htis logic out of the webview and into the dbtProject so we get it from the dbtCoreIntegration instead?

Reply:
@mdesmet do you mean just this getting query template from configuration or whole logic by getting macro from dbt as well? if it is just the config, I am not sure what is the benefit of moving it to core integration, because this is just getting from config file. If it is about whole logic of macro, that is not possible because it expects a query and limit as arguments which is not available in this context. pls lmk what you think.